package.json

```diff
@@ -29,26 +29,26 @@
     "url": "https://github.com/fijijavis/wdio-mochawesome-reporter/issues"
   },
   "dependencies": {
-    "@wdio/reporter": "^5.7.8",
-    "strip-ansi": "^5.2.0",
-    "uuid": "^3.0.1"
+    "@wdio/reporter": "^7.19.5",
+    "strip-ansi": "^6.0.0",
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "eslint": "^4.4.0",
-    "eslint-config-standard": "^10.2.0",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-node": "^5.1.0",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "jest": "^24.8.0",
+    "eslint": "^7.32.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.2.0",
+    "eslint-plugin-standard": "^5.0.0",
+    "jest": "^27.5.1",
     "jest-matchers": "^20.0.3"
   },
   "contributors": [
     "Jim Davis"
   ],
   "peerDependencies": {
-    "@wdio/cli": "^5.8.4",
-    "mochawesome-report-generator": "^3.1.5"
+    "@wdio/cli": "^7.19.5",
+    "mochawesome-report-generator": "^6.2.0"
```

Comment:
To make it work with 6.2.0, the output file format needs to be changed as well according to the contract https://github.com/adamgruber/mochawesome-report-generator/blob/master/src/bin/types.js

PS: I'm trying to make this lib work with wdio8 and the latest mochawesome-report-generator :)